appdatabase/migrations/sql/1671451982_add_balance_history.up.sql

```diff
@@ -0,0 +1,15 @@
+-- One entry per chain_id, address, currency, block
+-- bitset is used to select time interval granularity. The least significant bit has the coarsest granularity
+-- balance is a serialized big.Int
+CREATE TABLE IF NOT EXISTS balance_history (
+    chain_id UNSIGNED BIGINT NOT NULL,
+    address VARCHAR NOT NULL,
+    currency VARCHAR NOT NULL,
+    block BIGINT NOT NULL,
+    timestamp INT NOT NULL,
+    bitset INT NOT NULL,
+    balance BLOB
```

Comment:
This is more efficient and simple, and I have seen it done in other parts of the code for `big.Int`s. I can't see how this will improve other than human readability, which I didn't consider.

Reply:
The query should work with byte comparison (I can't see any case where it is required for balance, and we don't index it, though)